### DISCUSSION TITLE
I had to set the Ceph Orchestrator backend manually; is this normal?

### DISCUSSION THREAD

**Discussion by weeix** [rook/rook#11801] [General]
Hi. While I ran `ceph crash info <ID>` in the Rook Toolbox after deploying `CephCluster` and resolving some problems, I discovered the following message:

```
{
    "backtrace": [
        "  File \"/usr/share/ceph/mgr/nfs/module.py\", line 154, in cluster_ls\n    return available_clusters(self)",
        "  File \"/usr/share/ceph/mgr/nfs/utils.py\", line 38, in available_clusters\n    completion = mgr.describe_service(service_type='nfs')",
        "  File \"/usr/share/ceph/mgr/orchestrator/_interface.py\", line 1479, in inner\n    completion = self._oremote(method_name, args, kwargs)",
        "  File \"/usr/share/ceph/mgr/orchestrator/_interface.py\", line 1546, in _oremote\n    raise NoOrchestrator()",
        "orchestrator._interface.NoOrchestrator: No orchestrator configured (try `ceph orch set backend`)"
    ],
    "ceph_version": "17.2.5",
    "crash_id": "2023-02-28T08:48:56.650644Z_<REDACTED>",
    "entity_name": "mgr.a",
    "mgr_module": "nfs",
    "mgr_module_caller": "ActivePyModule::dispatch_remote cluster_ls",
    "mgr_python_exception": "NoOrchestrator",
    "os_id": "centos",
    "os_name": "CentOS Stream",
    "os_version": "8",
    "os_version_id": "8",
    "process_name": "ceph-mgr",
    "stack_sig": "<REDACTED>",
    "timestamp": "2023-02-28T08:48:56.650644Z",
    "utsname_hostname": "rook-ceph-mgr-a-<REDACTED>",
    "utsname_machine": "x86_64",
    "utsname_release": "4.18.0-425.3.1.el8.x86_64",
    "utsname_sysname": "Linux",
    "utsname_version": "#1 SMP Wed Nov 9 20:13:27 UTC 2022"
}
```

As a result, I had to configure the Ceph Orchestrator backend using the following [commands](https://serverfault.com/a/1063339/310145) in Rook Toolbox:

```
ceph mgr module enable rook
ceph orch set backend rook
```

Is this normal? I received a `TOO_FEW_OSDS` warning at first, so perhaps that's why this happened?

**Comment by microyahoo** [comment]
Do you want to create and manage nfs exports, right?  You can reference the document https://rook.io/docs/rook/v1.10/Storage-Configuration/NFS/nfs/#using-the-ceph-cli

**Reply by weeix** [reply]
Thanks. I'll keep this in mind when I need to manage nfs exports.

**Comment by travisn** [comment]
The rook mgr module is not enabled by default so this was expected, @jmolmo was investigating that.

**Reply by weeix** [reply]
I appreciate you confirming this. This is my first time deploying Rook Ceph, so I was a little alarmed when the manager pod crashed.

**Comment by MateuszBeru** [comment]
Problem persisted in version rook/ceph:v1.11.9 and I found it hard way when one of nodes went down due disaster and I was grated with an error during remove procedure
```
root@k8sPoC1 ~ # ceph orch host drain k8spoc3
Error ENOENT: No orchestrator configured (try `ceph orch set backend`)
```
